web/satellite/package-lock.json

```diff
@@ -82,6 +82,11 @@
         "webpack-bundle-analyzer": "4.5.0"
       }
     },
+    "../eslint-storj": {
```

Comment:
The changes in package-lock seem unrelated to the task at hand.

Note, do not update things unless the new code has been manually reviewed. See https://github.com/storj/docs/blob/static-service-ports/code/Dependencies.md